src/node/node.yml

```diff
@@ -1,4 +1,4 @@
-# Orb Version 0.1.0
+# Orb Version 0.2.0
```

Comment:
I think this counts as a major version bump

Reply:
I'm not sure that it will... let's just bump it to 1.0.0

Reply:
```suggestion
# Orb Version 1.0.0
```